### PR TITLE
fix(3928): remove exportTask validation via hitting the queryEndpoint. Use heuristic instead.

### LINKED ISSUE #3928: Notebook(Alerts): Export notebook as task triggers phantom run not recorded in task run history
## About the bug

**Steps to reproduce:**
List the minimal actions needed to reproduce the behavior.

1. Setup an echo listener to receive HTTP messages. Headers and body should be simply echoed to the console once authentication succeeds. 
2. Go to Influxdbv2 UI (note that during this test the feature flag 'notebooksNewEndpoints' was set)
3. Add test data to a bucket
4. Create an Alert Notebook
5. Use the test data to define the query
6. In the alert section choose HTTP
7. Add the URL to the listener
8. Click 'Export Alert Task'
9. Inspect the endpoints.  A check was run and notification was sent
10. Tasks > Open newly created task 

**Expected behavior:**

If a task was run immediately upon creation, expect a record of that run to be recorded in the "Runs" list.

**Actual behavior:**

The runs list is still empty. 

**Visual Proof:**

NBExportAlertAsTaskTriggersFantomRun01

## About your environment

**Environment info:**
Testing in K8SIDPE

latest commit project ui

```
commit 7f6a7997396b4cbae41cb300cafaac4306e73b56 (HEAD -> master, upstream/master, origin/master, origin/HEAD)
Author: wiedld <wiedld@users.noreply.github.com>
Date:   Fri Feb 18 16:40:49 2022 -0800
```

latest commit project K8SIDPE

```
commit 39c95a0c4ff722649d49c14bef5eb5f5726afebf (HEAD -> master, origin/master, origin/alchemy/automatic, origin/HEAD)
Author: Hercules Mango Churchill <alchemy@influxdata.com>
Date:   Sat Feb 19 01:41:48 2022 +0000
```

### PR DESCRIPTION
**Pull request by wiedld**
Closes #3928 

### Issue:
* we had a previous issue (https://github.com/influxdata/ui/pull/3916) of invalid Alert Tasks being produced.
     * We solved this by hitting the query endpoint first (which gives us errors if the flux is bad), BEFORE doing the crud call to export the task.
* Now we have a new ticket #3928 , about the running of the phantom query.

### Done:
* an imperfect compromise.
* we do not have semantic validation available yet
     * still TBD here https://github.com/influxdata/ui/issues/3926
* in the meantime:
    * remove phantom query (no hitting of the query endpoint)
    *  do a heuristic check of our most common semantic errors.

### Visual:
You will see:
* attempted export of bad query:
   * the query endpoint does not get triggered. (only the metrics collection API call)
   * an error notification is still surfaced to the user.
* fix it, and export a correct query: 
   * no error.
   * successfully exports the task.

https://user-images.githubusercontent.com/10232835/156840186-bce4738d-a227-461e-b6d3-9fded44d9089.mov

### CODE REVIEW

File: src/flows/pipes/Notification/ExportTask.tsx [hunk 1]
```diff
@@ -267,10 +276,13 @@ const ExportTask: FC = () => {
...
 
-  const validateTask = async (queryText: string): Promise<boolean> => {
+  const validateTask = (queryText: string): boolean => {
     try {
       setStatus(RemoteDataState.Loading)
-      await query(queryText)
+      // TODO: use full sematic validation, once this is done:
+      // https://github.com/influxdata/ui/issues/3926
+      // temporary check:
+      heuristicValidityCheck(queryText)
```
Comment (drdelambre) [inline src/flows/pipes/Notification/ExportTask.tsx:285]: Please remove. We can wait for the LSP updates if this is all we are testing

**Review by drdelambre — CHANGES_REQUESTED** [review]
we have a long history of generating issues that block production by doing this exact type of sting validation of queries within our interfaces. please remove.

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 53 | :red_circle:
accessibility | 97 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=3994)

File: src/flows/pipes/Notification/ExportTask.tsx [hunk 2]
```diff
@@ -27,10 +28,16 @@ import {
   exportAlertToTaskFailure,
 } from 'src/shared/copy/notifications'
 
+const validThresholds = (thresholds: Threshold[]) => {
+  return thresholds.every(
+    t => !!t.type && !!t.field && (t.value == 0 || !!t.value)
```
Comment (drdelambre) [inline src/flows/pipes/Notification/ExportTask.tsx:33]: these aren't the rules for thresholds or threshold input

**Comment by wiedld** [comment]
Compromised reached. Boatie gets only the code changes he wants (per https://github.com/influxdata/ui/pull/4018), and I wrote up a ticket addressing the larger problem we have with this feature (https://github.com/influxdata/ui/issues/4020).

**Pull request closed without merging**